src/App.js

```diff
@@ -21,15 +21,20 @@ const StyledApp = styled.div`
 
 function App() {
   const [cookies, setCookie, removeCookie] = useCookies(["sessionToken"]);
+  const cookieName = "sessionToken";
```

Comment:
Why not pass the cookieName variable to useCookies() instead of repeating the value?

Reply:
make sure to declare the variable first too.